ui/app/adapters/secret-v2-version.js

```diff
@@ -67,28 +74,57 @@ export default ApplicationAdapter.extend({
     return this._url(backend, path);
   },
 
-  v2DeleteOperation(store, id, deleteType = 'delete') {
```

Comment:
cleaned up delete operations. Problems with reloading because model.reload() is a promise and it wasn't fufilling.